### PR TITLE
proofread

### PR DESCRIPTION
**Pull request by Evils-Devils**

### CODE REVIEW

File: 5-membership.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ Why this process?
 
 *The members create and maintain the hacker environment.*
 
-When a conflict/problem can not be resolved between individuals/via do-ocracy or when it impacts the group, a group decision is required. Any member can request that a decision is made by the Group Of Members instead of by do-ocracy/individual members.
+When a conflict/problem can not be resolved between individuals, via do-ocracy or when it impacts the group, a group decision is required. Any member can request that a decision is made by the group of members instead of by do-ocracy or individual members.
```
Comment (galgalesh) [inline 5-membership.md]: "group of members" language isn't used anymore in the document. I just call it "meetings" now.

File: 6-guidelines.md [hunk 2]
```diff
@@ -51,15 +51,15 @@ There is a clear distinction between "personal" and "space" projects. Keep this
...
 
 #### Damaging or losing space property
 
-If you damage or lose space property, you have to notice the community immediately in the Changelog or the Main channel on mattermost. Explain what happened and if/how you will fix it.
+If you damage or lose space property, you have to notice the community immediately in the Changelog or Main channel on mattermost. Explain what happened and if/how you will fix it.
 
 #### Taking space property out of the space
 
-Only members are allowed to take space property out of the space. If you take space property out of the space, you have to notify the community immediately, for example in the "changelog" or "main" channels on Mattermost. If a somebody disagrees with you taking it out of the space, make that person happy or put it back.
+Only members are allowed to take space property out of the space. If you take space property out of the space, you have to notify the community immediately, for example in the Changelog or Main channel on Mattermost. If a someone disagrees with you taking it out of the space, make that person happy or put it back.
```
Comment (galgalesh) [inline 6-guidelines.md]: a someone -> someone ?

File: 5-membership.md (hunk 1 above)
Comment (galgalesh) [reply 5-membership.md]: so "is made on a meeting instead of by do-ocracy"

File: 6-guidelines.md (hunk 2 above)
Comment (Evils-Devils) [reply 6-guidelines.md]: somebody -> someone

Comment (galgalesh) [reply 6-guidelines.md]: Please change "If a somebody disagrees" to "if someone disagrees". You left the "a" there, which shouldn't be there imo.

**Pull request merged**